### ISSUE TITLE
Include meta?

### ISSUE THREAD

**Issue by drewclauson** [steffenbrem/JsonApiBundle#14]
How do I include meta in my response?

I was able to get it to work with the following:

```
return array("data"=>$data, "meta"=>array("test"=>50));
```

However, this was removed from the response when I did so:

```
{
  "jsonapi": {
    "version": "1.0"
  }
}
```

and it also added a blank included array (unless I had side loaded objects, then it correctly side loaded them):

```
{ 
   "included": [],
...
}
```

**Comment by koemeet** [comment]
@drewclauson I will begin working on a overriding data and meta. Later I wanted to introduce a metadata  listener that you can use for every resource type. It will be invoked for every resource or for a collection and you can return in that listener whatever you want. It will then have the resource object etc. as an argument. This way you have the metadata bound to the resource and everything that includes that resoruce will include the same metadata. But that is something for later, I will begin with implementing your example, I hope to get that working today, will open a PR for it.

**Comment by koemeet** [comment]
I have merged the PR #15 into master :+1:  Thanks!

**Comment by drewclauson** [comment]
Thanks! I didn't know if there was a better or a "right" way to do it. I did, however dive into the code yesterday and have a better understanding of how your bundle works. I'm still pretty fresh to Symfony/rest/fos/ember, but I'm hoping to make some good contributions.
